assets/src/edit-story/components/canvas/selectionCanvas.js

```diff
@@ -150,8 +150,8 @@ function SelectionCanvas({ children }) {
   const onMouseUp = () => {
     if (lassoModeRef.current === LassoMode.ON) {
       const [lx, ly, lwidth, lheight] = getLassoBox();
-      const x = editorToDataX(lx - pageContainer.offsetLeft);
-      const y = editorToDataY(ly - pageContainer.offsetTop);
+      const x = editorToDataX(lx - fullbleedContainer.offsetLeft);
```

Comment:
Hmm. I still don't get it. I'd expect all the coordinate system to work with pageContainer, not fullbleedContainer. LEt's discuss.

Reply:
Discussed offline